lib/bloc/backup/backup_bloc.dart

```diff
@@ -44,10 +44,12 @@ class BackupBloc {
       BehaviorSubject<BackupState>();
   Stream<BackupState> get backupStateStream => _backupStateController.stream;
 
-  final StreamController<bool> _promptBackupController =
-      StreamController<bool>.broadcast();
+  final StreamController<bool> _promptBackupController = BehaviorSubject<bool>.seeded(false);
   Stream<bool> get promptBackupStream => _promptBackupController.stream;
 
+  final StreamController<bool> _promptBackupDismissedController = BehaviorSubject<bool>.seeded(false);
```

Comment:
Should the behavior of "Don't prompt again" be persistent?

Reply:
It is to avoid https://github.com/breez/breezmobile/issues/1116, the same reason I'm changing line 47 too